README.md

```diff
@@ -20,29 +20,19 @@ Conduct the following process for each network:
 - Perform a dry-run and confirm that the actions that would be executed by Cannon are expected:
 
   ```sh
-  yarn cannon build omnibus-base-sepolia-andromeda.toml \
-    --dry-run \
-    --upgrade-from synthetix-omnibus:latest@andromeda \
-    --chain-id 84532 \
-    --provider-url https://base-sepolia.infura.io/v3/$INFURA_API_KEY
+  yarn cannon build omnibus-base-sepolia-andromeda.toml --dry-run --upgrade-from synthetix-omnibus:latest@andromeda --chain-id 84532 --provider-url https://base-sepolia.infura.io/v3/$INFURA_API_KEY
```

Comment:
I don't agree this is a good change.

Reply:
I replaced pieces of the example code and forgot to add a trailing backslash with resulted in mistaken behavior. I think it's fine to reduce readability here.